src/libraries/System.Security.Cryptography/tests/X509Certificates/PfxTests.cs

```diff
@@ -561,33 +561,103 @@ public static void ReadMLKem1024PrivateKey_Both_Pfx(X509KeyStorageFlags keyStora
         [MemberData(nameof(StorageFlags))]
         public static void ReadMLKem512PrivateKey_NotSupported(X509KeyStorageFlags keyStorageFlags)
         {
-            byte[] pfxBytes = MLKemTestData.IetfMlKem512PrivateKeySeedPfx;
-            string pfxPassword = MLKemTestData.EncryptedPrivateKeyPassword;
+            const string PfxPassword = "PLACEHOLDER";
```

Comment:
Ah, right.  Since a password is provided X509Certificate2..ctor is passing DangerousNoLimits, bypassing our higher level decryption that we'd do in the loader.

"Yay, compat"